cmake/CMakeLists.txt

```diff
@@ -1338,102 +1338,105 @@ if (onnxruntime_ENABLE_TRAINING)
   add_compile_definitions(ENABLE_STRIDED_TENSORS)
   add_compile_definitions(ENABLE_TRAINING)
 
-  if (UNIX)
-    if (EXISTS "${onnxruntime_MPI_HOME}")
-      set(MPI_HOME "${onnxruntime_MPI_HOME}")
-    elseif (EXISTS "/bert_ort/openmpi")
-      set(MPI_HOME "/bert_ort/openmpi")
-    endif()
+  add_subdirectory(tensorboard EXCLUDE_FROM_ALL)
```

Comment:
What is tensorboard used for? Can you add a comment?

Reply:
it is not my change, it existed in the current code, i just move the mpi related part out of training config.